feat: add form data handling and file handling (#778)
sparckles/Robyn: docs_src/src/pages/documentation/api_reference/form_data.mdx

export const description =
  'On this page, we’ll dive into using the form data.'

## Fom Data

Batman learned how to handle file uploads using Robyn. Now, he wanted to handle the form data

## Handling Form Data

<Row>
<Col>

> Batman uploaded an encoded form data and wanted to handle it using the following code:

Author: ```suggestion
Batman uploaded some multipart form data and wanted to handle it using the following code:
```
